### PR TITLE
feature(REPORT): Bold reports V5.4 Release Changes

### PR DESCRIPTION
**Pull request by Vishnu7101**
<table>
    <tr>
        <td>Feature description</td>
        <td>
To Update the Bold Reports - ASP.NET MVC public repo with the  Bold-Reports 5.4 Release Changes</td>
    </tr>
    <tr>
        <td>Analysis</td>
        <td> Analyzed about Bold reports V5.4 release changes</td>
    </tr>
    <tr>
        <td>Reason for not identifying earlier</td>
        <td>
            It's a new feature</td>
    </tr>
    <tr>
        <td>Is it a breaking issue?</td>
        <td>No, It's a new feature </td>
    </tr>
    <tr>
        <td>Solution description</td>
        <td>
<li>To modify the files according to the latest changes made in Bold report V5.4
<li>Included package josn and gulp file in order to download puppeteer.
</td>
    </tr>
    <tr>
        <td>Areas affected and ensured</td>
        <td>NA</td>
    </tr>
    <tr>
        <td>Test cases</td>
        <td>NA</td>
    </tr>
    <tr>
        <td>Test bed sample locations</td>
        <td>VS & VS Code</td>
    </tr>
</table>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -18,6 +18,12 @@ The samples requires the below requirements to run.
 * [Visual Studio 2019](https://visualstudio.microsoft.com/downloads/)
 * .Net Framework 4.5 and above
 
+In order to download samples in the Report Writer Properly , run the below mwntioned command.
```
Comment (Subhasree-Vasudevan) [inline README.md:21]: Can we change to the following?

To export the data visualization report items in report writer, run the below mentioned command. Without this, the data visualization report items no longer available in the report exported documents only for the report writer.

File: package.json [hunk 2]
```diff
@@ -0,0 +1,17 @@
+{
+  "name": "aspnet-mvc-samples",
+  "version": "5.4.20",
+  "author": "Syncfusion",
+  "license": "Syncfusion",
+  "scripts": {
+    "postinstall": "gulp Puppeteer-download"
+  },
+  "devDependencies": {
+    "gulp": "^4.0.2"
```
[the remark this answers is not in the document]
Comment (Vishnu7101) [reply package.json]: Updated

File: package.json [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+  },
+  "dependencies": {
+    "shelljs": "0.8.5",
+    "unzipper": "^0.10.11",
```
[the remark this answers is not in the document]
Comment (Vishnu7101) [reply package.json]: Updated

File: gulpfile.js [hunk 4]
```diff
@@ -0,0 +1,21 @@
+var gulp = require("gulp");
+const download = require("gulp-download");
+const { createReadStream } = require('fs');
+const { rm, mkdir } = require('shelljs');
+const unzipper = require('unzipper');
+var path = require('path');
+const puppeteerPath = 'Scripts/puppeteer/Win-901912';
+
+gulp.task('Puppeteer-download', (done) => {
```
[the remark this answers is not in the document]
Comment (Vishnu7101) [reply gulpfile.js]: Updated

File: README.md (hunk 1 above)
Comment (Vishnu7101) [reply README.md]: Updated

**Review by Subhasree-Vasudevan — APPROVED** [review]
Changes Fine

File: Content/css/bold-reports/bold.reportdesigner.core.material.min.css [hunk 5]
```diff
@@ -0,0 +1,11 @@
+﻿@charset "UTF-8";
```
[the remark this answers is not in the document]
Comment (Vishnu7101) [reply Content/css/bold-reports/bold.reportdesigner.core.material.min.css]: Removed

**Review by karthickthangasamy — APPROVED** [review]

**Pull request merged**